e2e/testconfig/testconfig.go

```diff
@@ -8,27 +8,31 @@ import (
 )
 
 const (
-	// ChainASimdImageEnv specifies the image that Chain A will use.
-	ChainASimdImageEnv = "CHAIN_A_SIMD_IMAGE"
-	// ChainASimdTagEnv specifies the tag that Chain A will use.
-	ChainASimdTagEnv = "CHAIN_A_SIMD_TAG"
-	// ChainBSimdImageEnv specifies the image that Chain B will use. If unspecified
+	// ChainImageEnv specifies the image that the chains will use. If left unspecified, it will
+	// default to being determined based on the specified binary. E.g. ghcr.io/cosmos/ibc-go-simd
+	ChainImageEnv = "CHAIN_IMAGE"
+	// ChainATagEnv specifies the tag that Chain A will use.
+	ChainATagEnv = "CHAIN_A_TAG"
+	// ChainBTagEnv specifies the tag that Chain B will use. If unspecified
 	// the value will default to the same value as Chain A.
-	ChainBSimdImageEnv = "CHAIN_B_SIMD_IMAGE"
-	// ChainBSimdTagEnv specifies the tag that Chain B will use. If unspecified
-	// the value will default to the same value as Chain A.
-	ChainBSimdTagEnv = "CHAIN_B_SIMD_TAG"
+	ChainBTagEnv = "CHAIN_B_TAG"
 	// GoRelayerTagEnv specifies the go relayer version. Defaults to "main"
 	GoRelayerTagEnv = "RLY_TAG"
-	// ChainBinary binary is the binary that will be used for the chains.
-	ChainBinary = "CHAIN_BINARY"
+	// ChainBinaryEnv binary is the binary that will be used for both chains.
+	ChainBinaryEnv = "CHAIN_BINARY"
 	// defaultBinary is the default binary that will be used by the chains.
 	defaultBinary = "simd"
-	// defaultSimdImage is the default image that will be used for the chain if none are specified.
-	defaultSimdImage = "ghcr.io/cosmos/ibc-go-simd"
-	defaultRlyTag    = "main"
+	// defaultRlyTag is the tag that will be used if no relayer tag is specified.
+	defaultRlyTag = "main"
 )
 
+func getChainImage(binary string) string {
+	if binary == "" {
+		binary = defaultBinary
+	}
+	return fmt.Sprintf("ghcr.io/cosmos/ibc-go-%s", binary)
```

Comment:
Interesting. I wonder how useful this will be outside of this context.

Reply:
I don't think outside of this context there would be much use for this. The main thing is that we can always specify `CHAIN_IMAGE` to overwrite it.

Reply:
Yea that's what I was thinking. I guess we won't be able to remove it until we stop supporting v4